parse-args.js

```diff
@@ -208,6 +208,10 @@ function yargsParser (argv, defaultNpm) {
       describe: Y()`Ignores existing binaries in $PATH, or in the local project. This forces npx to do a temporary install and use the latest version.`,
       type: 'boolean'
     })
+    .option('in-process', {
+      describe: Y()`Allow the command to be executed in the same process as npx instead of a child process.`,
+      type: 'boolean'
```

Comment:
in re opt-in, I think this should default to `true` -- that way, it won't be a breaking change, and this has been part of npx for a while, so I'd rather not start spawning processes here.

Reply:
What do you think about flipping the new option to being `child` which enforces the child process branch? I kind of dig that approach since the `in-process` option is more of a suggestion if it's doable _(no `nodeArg` or `shell` or other stuff)_ where the `child` option is explicit and can happen 💯of the time.